tests/testdata/cass-operator-1.1.0-chart/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v2
```

Comment:
I need access to the 1.1.0 helm chart somehow. It was easiest to just include it in the testdata. Open to suggestions.

Reply:
we could get it with git, but this is fine with me